synthtool/README.md

```diff
@@ -0,0 +1,386 @@
+# SynthTool (for client libraries)
+This tool helps to generate and layout cloud client libraries. Synthtool runs the [GAPIC (Generated API Client) Generator][GAPIC] via [Google API Artifact Manager (artman)][artman].
+
+[GAPIC]: https://github.com/googleapis/gapic-generator
+[artman]: https://github.com/googleapis/artman
+
+## Prerequisites
+
+1. **Python 3.6** Either install it from [python.org][python_downloads] or use
+[pyenv][] to get 3.6.
+
+2. **Bazel**  can be downloaded from [bazel.build](https://bazel.build/).
+
+3. **Docker**  Some synth.py files require [Docker] to generate code.
+
+[python_downloads]: https://www.python.org/downloads/
+[pyenv]: https://github.com/pyenv/pyenv
+[Docker]: https://docs.docker.com/v17.09/engine/installation/#desktop
+
+## Add synthtool to your PYTHONPATH
+
+```
+..\synthtool$ export PYTHONPATH=`realpath .`
```

Comment:
What's the ..\ here?

Reply:
It's intended to be read as an ellipsis:
some-path-on-your-computer\synthtool$

Reply:
Not obvious, I'm afraid.  I think just `$` is what most tech docs use in this context. Not everyone has their bash prompt set to show the directory.

Reply:
I compared to the other commands in this doc, and they don't even have a $.  So I removed ..\synthtool$ to make it consistent.